### ISSUE TITLE
Improvement Suggestion: Output Empty Mask Overlay When No Objects Detected

### ISSUE THREAD

**Issue by ro-hit81** [opengeos/segment-geospatial#348]
https://github.com/opengeos/segment-geospatial/blob/8888ba6471cedc22ba790601b058cd33d5544257/samgeo/text_sam.py#L343

**Context**: In the current implementation, when no object instances are found in the image, the function returns early after printing "No objects found in the image."

**Proposed Change**: I suggest adding a line to return an empty mask overlay after the print statement. This can be beneficial for tasks where a mask overlay is necessary, even when no objects are detected.

```
if boxes is None or (len(boxes) == 0):  # No "object" instances found
    print("No objects found in the image.")
    mask_overlay = np.zeros_like(image_np[..., 0], dtype=dtype)  # Create an empty mask overlay
```
 
**Rationale**: This change ensures that users can still work with the function's output consistently, even in cases where no objects are found, allowing for easier downstream processing and integration.

**Comment by giswqs** [comment]
Good suggestion. Would you like to submit a pull request? Contrubtions are welcome.

**Comment by ro-hit81** [comment]
Done.

**Comment by giswqs** [comment]
Fixed in #350
